compose.dev.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3.8'
+services:
+  api:
+    build:
+      context: ./api
+      dockerfile: Dockerfile.dev
+    develop:
+      watch:
+        - action: sync
+          path: ./api
+          target: /app
+        - action: rebuild
+          path: ./api/Cargo.toml
+    restart: unless-stopped
+    depends_on:
+      - database
+    ports:
+      - 8080:8080
```

Comment:
Same comment as for the compose.yam file, just use the same port forwarding layout, and if you want specific ports just use environment variables